electronics/charging_capacitor.py

```diff
@@ -0,0 +1,114 @@
+# source - The ARRL Handbook for Radio Communications
+
+from math import exp  # value of exp = 2.718281828459…
+
+"""
+Description
+-----------
+when a capacitor is connected with a potential source(AC or DC). It is starts to charge
+at a general speed but when a resistor is connected in the  circuit with in series to
+a capacitor then the capacitor charges slowly means it will take more time than usual.
+while the capacitor is being charged, the voltage is in exponential function with time.
+
+in the this function there is RC which is 'resistance(ohms)*capacitance(farads)'.
+also represented as τ (tau).
+
+with the help of RC-timeconstant we can find the voltage at any time 't' from the
+initiation of charging a capacitor with the help of the exponential function
+containing RC.Both at charging and discharging of a capacitor.
+"""
+
+
+def charging_capacitor(
+    source_voltage: float,  # voltage in volts.
+    resistance: float,  # resistance in ohms.
+    capacitance: float,  # capacitance in farads.
+    time_sec: float,  # time in seconds after charging initiation of capacitor.
+):
```

Comment:
This function should never return `None`.  It should return a `voltage` or raise an exception if the voltage cannot be calculated.
```suggestion
) -> float:
```

Reply:
DONE!